.config/nextest.toml

```diff
@@ -11,5 +11,18 @@ retries = 5
 # Timeout tests after 4 minutes
 slow-timeout = { period = "60s", terminate-after = 4 }
 
+[profile.ignored]
```

Comment:
Maybe call this `profile.nightly`? `ignored` seems like they should never run, which confused me in narwhal before.

Reply:
Good point - will do